### ISSUE TITLE
The positions of full-image pics seemed to be shifted

### ISSUE THREAD

**Issue by jcf94** [iissnan/hexo-theme-next#1911]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.iissnan.com/).
- [x] And yes, I already searched for current [issues](https://github.com/iissnan/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
full-image style pics align centered in the page

### Actual behavior 
full-image style pics shift from the center

### Steps to reproduce the behavior 
1. Add `<img src="xxxx" class="full-image"/>` above the `<!--more-->` tag
2. Pics are shifted in the index as well as in the post page
3. But the full-image pic in my about page present to be right

* Link to demo site with this issue: http://jcf94.com/
* Link(s) to source code or any usefull link(s): N/A

### NexT Information

**NexT Version:**

```
[x] Latest Master branch.
[] Latest Release version.
[] Old version - 
```

**NexT Scheme:**

```
[] All schemes
[] Muse
[] Mist
[] Pisces
[x] Gemini
```

### Other Information 
This error shows in the latest Chrome & Edge Browser.

The shifted pic:
![](http://7xnyi3.dl1.z0.glb.clouddn.com/error1.png)

The right pic in my about page:
![](http://7xnyi3.dl1.z0.glb.clouddn.com/error2.png)

Maybe this error is caused by the position of fancybox? I just notice that the right pic in my about page is actually a js div, and the shifted pic in my index page and post page is inside a fancybox.

**Comment by ivan-nginx** [comment]
Please, try to follow next steps:

1. Backup your source of `themes/next` directory.
2. Switch to latest release version (see `Download latest master branch.` section in main readme).
3. Try to catch same error by refreshing browser with `CTRL` + `SHIFT` + `R`.

**Comment by jcf94** [comment]
I've done these steps & my blog is using the latest mater branch now.
Chrome did not catch any error.
![](http://7xnyi3.dl1.z0.glb.clouddn.com/error3.png)

**Comment by jcf94** [comment]
Hi, I guess I find out where the problem is.
See this:
[http://7xnyi3.dl1.z0.glb.clouddn.com/error4.png](http://7xnyi3.dl1.z0.glb.clouddn.com/error4.png)
And this:
[http://7xnyi3.dl1.z0.glb.clouddn.com/error5.png](http://7xnyi3.dl1.z0.glb.clouddn.com/error5.png)

**Comment by ivan-nginx** [comment]
I agree what this must be fixed, but i think need to fix full-image totally. Originally, it was like on your right (last) screen, +125% width and centered. On other schemes as i remember this look same. A main feature of this was di full-image style like «convert» style. Like this picture scrolled-in whole post. I think need to do it like this or do it simple, wrapped in post like simple image (but mb on all width).

**Comment by ivan-nginx** [comment]
I think best live demo of image «convert» style on [Jekyll docs site](https://jekyllrb.com/docs/frontmatter/).  See on red and yellow info tips: they like convert out of main content wrap.

**Comment by geekrainy** [comment]
Fixed #2043

**Comment by ivan-nginx** [comment]
Merged.
If solved, close it please.

**Comment by jcf94** [comment]
Closed, and I'm going to try your 6.0 version.
